apps/frontend/src/components/global/upload_tabs/HelpFooter.vue

```diff
@@ -22,6 +22,14 @@
           <v-icon small>mdi-github</v-icon>
           <span class="d-none d-sm-inline pl-3">Github</span>
         </v-btn>
+        <v-btn href="https://www.netlify.com/" target="_blank" text small>
+          <img
+            :src="require('@/assets/netlify-light.svg')"
+            style="width: 1.15rem; height: 1.15rem"
+            alt="Netlify.com"
+          />
+          <span class="d-none d-sm-inline pl-3">Netlify</span>
+        </v-btn>
```

Comment:
can you change the order of the buttons to look as follows:

about, help, docs, github, netlify, release+version